src/fetcher/query.ts

```diff
@@ -311,6 +311,8 @@ export const dataQuery = gql`
             }
             ... on ExerciseGroup {
               id
+              alias
```

Comment:
wow, I screwed up something, this is the way I tried first but it did not work.
thanks for the cleaner solution 👍